### ISSUE TITLE
Inconsistent pairing between Zigbee2MQTT and STM32

### ISSUE THREAD

**Issue by Gunlek** [rubenmiddelman/STM32_to_Zigbee#2]
Hello,

I've described an issue I'm facing here: https://github.com/home-assistant/core/issues/111948
TL;DR:
I cannot repeat a consistent pairing scheme between STM32WB5MM-DK and Zigbee2MQTT. If I remove the device from Z2M after a successful pairing, I have to wait an inconsistent amount of time before the device is paired successfully (or not at all)...

I looked at the logs (ZHA before moving to Zigbee2MQTT, PCAP using wireshark and zigbee sniffer) but there is no specific failed packet...

I wonder if you faced the same kind of issue while developing this project as STM32WB5MM-DK is based -- obviously -- on STM32WB5MM which is the same as the nucleo board you used.

I can collect and provide any logs if required for further investigation
